WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift

```diff
@@ -404,6 +437,12 @@ class PluginViewModel: Observable {
         return alert
     }
 
+    private func routeToRegisterDomain() {
```

Comment:
Maybe we can make this name more consistent. Something like `presentDomainRegistration` or `showDomainRegistration` perhaps?

Reply:
Done.